src/value.c

```diff
@@ -513,6 +513,10 @@ int SIValue_Compare(const SIValue a, const SIValue b, int *disjointOrNull) {
 		case T_BOOL:
 			return SAFE_COMPARISON_RESULT(a.longval - b.longval);
 		case T_DOUBLE:
```

Comment:
I saw you've changed in Register_BooleanFuncs the params to SI_ALL, aren't we missing here some types like: T_DATE?

Reply:
T_DATE is not used now